content/04.model-complexity.md

```diff
@@ -1,32 +1,33 @@
+Techniques like resampling and combining various ML methods together (ensemble learning) can help achieve stability in predictions (Figure[@fig:2]A-B). 
 Resampling without replacement can generate confidence intervals for the model predictions by iteratively exposing the models to incomplete datasets, mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
-Resampling with replacement, or bootstrapping, helps estimate population values from datasets of limited size, and is also commonly used to find robust models when multiple models are combined into an ensemble ([@doi:10.1080/01621459.1997.10474007; @doi:10.1023/A:1010933404324; @doi:10.1198/0003130043277; @doi:10.1016/s0925-2312(01)00650-6; @doi:10.1016/j.neucom.2004.11.017; @doi:10.1002/sim.4780111607]).
-Ensemble learning methods like random forests use _bagging_ (bootstrap aggregation) of independent decision trees that use similar parameters but different paths to form a consensus about the important predictive features [@doi:10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134; @doi:10.1214/aos/1031689014; @doi:10.1177/2045894019890549; @doi:10.1016/s0031-3203(02)00169-3]. 
-Regular ensemble learning has shown limited success in rare disease datasets with substantial label-noise. [TODO: citation?] 
-This has led to the adoption of cascade learning, a variant of ensemble learning, where multiple methods leveraging distinct underlying assumptions are used in tandem; and augmented with algorithms like AdaBoost (boosting) to capture stable patterns existing in the silver standard data [@doi:10.1109/cvpr.2001.990537; @doi:10.1007/978-3-540-75175-5_16; @doi:10.1109/icpr.2004.1334680]. 
-A cascade learning approach for identifying rare disease patients from electronic health records from the general population utilized independent steps for feature extraction (using natural language processing based word2vec [@https://arxiv.org/abs/1301.3781v343]), preliminary prediction using an ensemble of decision trees, and prediction refinement using similarity of data points to resolve sample labels [@pmid:30815073]. 
+Alternatively, resampling with replacement ('bootstrapping') helps estimate population values from datasets of limited size, and is also commonly used to find robust models when multiple models are combined into an ensemble ([@doi:10.1080/01621459.1997.10474007; @https://doi.org/10.1023/A:1010933404324; @doi:10.1198/0003130043277; @doi:10/c8xpqz; @doi:10.1016/j.neucom.2004.11.017; @doi:10.1002/sim.4780111607]).
+Ensemble learning methods like random forests use _bagging_ (bootstrap aggregation) of independent decision trees that use similar parameters but different paths to form a consensus about the important predictive features [@https://doi.org/10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134; @doi:10.1214/aos/1031689014; @doi:10.1177/2045894019890549; @doi:10/btzfh6]. 
```

Comment:
Why does bootstrapping get quotes but bootstrap aggregation does not? Same question about ensemble learning.

Reply:
Upon reflection I think I prefer it with no quotes, so I changed this for bootstrapping. Maybe need to scrub the rest of the paper for this too....